presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++      ``task.info-refresh-max-wait``      +          ``String`` (duration)           +             ``200 ms``             +Controls staleness of task information which is used   +
++                                          +                                          +                                    +in scheduling. Increasing this value can reduce        +
++                                          +                                          +                                    +coordinator CPU load but may result in suboptimal      +
++                                          +                                          +                                    +split scheduling.                                      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.max-worker-threads``        +               ``Integer``                +    ``Number of node CPU's * 4``    +Sets the number of threads used by workers to process  +
++                                          +                                          +                                    +splits. Increasing this number can improve throughput  +
++                                          +                                          +                                    +if worker CPU utilization is low but will cause        +
++                                          +                                          +                                    +increased heap space usage.                            +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++      ``distributed-joins-enabled``       +               ``Boolean``                +              ``true``              +Use hash distributed joins instead of broadcast joins. +
++                                          +                                          +                                    +Distributed joins require redistributing both tables   +
++                                          +                                          +                                    +using a hash of the join key. This can be slower       +
++                                          +                                          +                                    +(sometimes substantially) than broadcast joins but     +
++                                          +                                          +                                    +allows much larger joins. Broadcast joins require that +
++                                          +                                          +                                    +the tables on the right side of the join fit in memory +
```

Comment:
This is one of three properties that were already described and I've only put them into the table without really looking into them to much.
But sure, to be done.